src/main/java/decodes/cwms/resevapcalc/ResEvap.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (c) 2018
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+package decodes.cwms.resevapcalc;
+
+import decodes.tsdb.CTimeSeries;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+/**
+ * Program to estimate evaporation from reservoirs
+ * Program estimates 1-D water temperature profile in reservoir
+ * Calculates evaporation
+ *
+ * @author RESEVAP program by Steven F. Daly (ERDC/CRREL)
+ * Version 1.1  31 August 2009
+ * VERSION 2.0 7 July 2010
+ * conversion to Java by Richard Rachiele (RMA)
+ */
+public class ResEvap {
+    private static final Logger LOGGER = Logger.getLogger(ResEvap.class.getName());
+
+    // Some global constant parameter vaiues set here
+    public static final double EMITTANCE_H20 = 0.98;
+    public static final double PENFRAC = 0.4;
+    public static final double ALBEDO = 0.08;
+    public static final double THETA = 1.0;
+    public static final double SIGMA = 5.67e-8;
+    public static final double ETA_CONVECTIVE = .50;
+    public static final double ETA_STIRRING = .40;
+    public static final double WIND_CRITIC = 1.0;
+
+    // output computed time series
+    double solarRadTsc;
```

Comment:
It looks like many of these variables can be made private.  Or is any of this being accessed with reflection?